docs/resources/compute_instance.md

```diff
@@ -529,7 +529,7 @@ terraform import huaweicloud_compute_instance.my_instance b11b407c-e604-4e8d-8bc
 
 Note that the imported state may not be identical to your resource definition, due to some attributes missing from the
 API response, security or some other reason.
-The missing attributes include: `admin_pass`, `user_data`, `metadata`, `data_disks`, `scheduler_hints`, `stop_before_destroy`,
+The missing attributes include: `admin_pass`, `metadata`, `data_disks`, `scheduler_hints`, `stop_before_destroy`,
```

Comment:
the value of the parameter `user_data` is never save in the Read function, why we remove it?

Reply:
I misunderstood and have already corrected it back.